app/services/recurring_contribution/subscriptions/processor.rb

```diff
@@ -1,23 +1,26 @@
 class RecurringContribution::Subscriptions::Processor
-  attr_reader :subscription
+  attr_reader :subscription, :with_save
 
-  def initialize(subscription, credit_card_hash)
+  def initialize(subscription, credit_card_hash, with_save)
```

Comment:
@talyssonoc you're totally right. I'll change this, I was not concerned about the calling of the initialize because I use this service calling directly the self.process method. You can see that in this method I have a named parameter for the `with_save`. Thanks!